drivers/sensor/iis2iclx/Kconfig

```diff
@@ -53,20 +53,6 @@ config IIS2ICLX_THREAD_STACK_SIZE
 	help
 	  Stack size of thread used by the driver to handle interrupts.
 
```

Comment:
Shouldn't there be a `default y if $(dt_compat_enabled,...)` in the top level menuconfig now, since you're here?

Reply:
Can you be more specific. I'm not getting the point, my bad. :(

Reply:
Hmm, never mind. I think I am misunderstanding the results of a separate discussion. Sorry for noise, I've taken the discussion to the devicetree channel on slack for now.